mendix/pusher-notify-listen
# [WC-2031] Add migration file

Add a migration file for upcoming Java deps management.

## How to test

Download mx 10.2
Enable JDM by using `--enable-jar-dependency-management`
Import Pusher module

Expected:
- `userlib` should be empty.
- App Explorere>Pusher>Settings - you should see the pusher library as a dependency.
- project build should have no errors.
